code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -1940,4 +1940,28 @@
 	name = "Nanoblood"
 	result = /datum/reagent/nanoblood
 	required_reagents = list(/datum/reagent/dexalinp = 1, /datum/reagent/iron = 1, /datum/reagent/blood = 1)
-	result_amount = 3
\ No newline at end of file
+	result_amount = 3
+	
+/datum/chemical_reaction/amnestics/a
+	name = "Amnestics-A"
+	result = /datum/reagent/amnestics/a
+	required_reagents = list(/datum/reagent/ethanol = 20, /datum/reagent/water = 10, /datum/reagent/sugar = 10)
```

Comment:
Doesn't this look way too simple and not too realistic? I suggest we use materials unavailable in every household or present times, such as phoron in the recipes!

Reply:
That would need a massive amount of phoron every round and unnecessary tbh. I'm open to discussion though.

Reply:
I don't say exactly phoron, just sugar, alcohol and water should not create basically anomalous reagents imho

Reply:
The recipe might change over time, but for now, this is what it is. I think we've got a good base here. Unless @Affurit wishes to suggest changes?